frontend/src/Crossword.tsx

```diff
@@ -441,7 +436,7 @@ const Crossword: React.FC<CrosswordProps> = (props) => {
         width={props.image.width}
         height={props.image.height}
         ref={backgroundRef}
-        className="crossword"
+        className="crossword background"
```

Comment:
I don't think there is a background class anywhere

Reply:
This just makes it easier to distinguish which canvas is for the background and which one is for the drawing layer when browsing the elements in the developer panel in chrome/firefox/edge :D

Reply:
Could you name it something more descriptive (`background-noop`?)

Reply:
I just removed it. the `draw-layer` class in the bottom canvas should be enough